Packs/JoeSecurity/Playbooks/playbook-Detonate_File_-_JoeSecurity_V2.yml

```diff
@@ -372,11 +372,20 @@ tasks:
       - - operator: match
           left:
             value:
-              simple: File.Type
+              simple: inputs.File.Type
+            iscontext: true
+          right:
+            value:
+              simple: .*(?:EXE|DLL|PIF|CMD|BAT|COM|SCR|CPL|PDF|DOCX|DOC|DOCM|XLSX|XLS|XLSM|XLSB|PPT|PPX|PPTM|HWP|JTD|RFT|XPI|CRX|EML|MSG|CHM|JS|JSE|VBS|VBE|LNK|JAR|PS1|ZIP|7Z|RAR|ZLIB|ASPX|APS|PNG|JPEG|GIF|HTML|HTM|XHTML|SHTML|WORD)\b
```

Comment:
File type is without dot(".") please fix and do tests to make sure its working

Reply:
It is not a "."
. matches any character (except for line terminators)
* matches the previous token between zero and unlimited times, as many times as possible, giving back as needed